src/components/FileDropzone/MucFileDropzone.vue

```diff
@@ -0,0 +1,278 @@
+<template>
+  <div
+    class="drop-zone"
+    :class="{
+      'is-dragover': isDragOver,
+      'is-not-disabled': !disabled,
+    }"
+    @dragover.prevent="onDragOver"
+    @dragleave.prevent="onDragLeave"
+    @drop.prevent="onDrop"
+    @click="selectFiles"
+  >
+    <svg
+      aria-hidden="true"
+      width="108"
+      height="56"
+      viewBox="0 0 108 56"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+    >
```

Comment:
I would prefer to extract this svg either into a file that you can import or create a new component for it

Reply:
It is an external component now.